pydantic/main.py

```diff
@@ -1473,6 +1474,19 @@ def create_model(  # noqa: C901
                     'Field definitions should be a `(<type>, <default>)`.',
                     code='create-model-field-definitions',
                 ) from e
+
+        elif _typing_extra.is_annotated(f_def):
+            (f_annotation, f_value, *_) = typing_extensions.get_args(
+                f_def
+            )  # first two input are expected from Annotated
```

Comment:
Not sure what you mean by the comment here, but the logic looks right!

Reply:
I put this comment because we are assuming the first two inputs and only use the first two inputs. This comment is to give an explanation so anyone who sees the code doesn't have to read up on the syntax of the Annotated